articles/iot-edge/tutorial-configure-est-server.md

```diff
@@ -31,6 +31,9 @@ This tutorial walks you through hosting a test EST server and configuring an IoT
 * Your IoT Edge device requires Azure IoT Edge runtime 1.2 or later for EST support. Azure IoT Edge runtime 1.3 or later required for EST certificate renewal. 
 * IoT Hub Device Provisioning Service (DPS) linked to IoT Hub. For information on configuring DPS, see [Quickstart: Set up the IoT Hub Device Provisioning Service with the Azure portal](../iot-dps/quick-setup-auto-provision.md).
 
+> [!NOTE]
+> To use EST and IoT Edge for automatic device identity certificate issuance and renewal, which is recommended for production, IoT Edge must provision as part of a [DPS CA-based enrollment group](./how-to-provision-devices-at-scale-linux-x509.md?tabs=group-enrollment%2cubuntu). Other methods of provisioning, including manual X.509 provisioning with IoT Hub and DPS with individual enrollment do not support automatic device identity certificate renewal.
```

Comment:
The query string parameter isn't supported for file relative links. If you need the conceptual tab, you have to use a site relative link. If you use a query parameter for file relative links, it's ignored.

```suggestion
> To use EST and IoT Edge for automatic device identity certificate issuance and renewal, which is recommended for production, IoT Edge must provision as part of a [DPS CA-based enrollment group](./how-to-provision-devices-at-scale-linux-x509.md). Other methods of provisioning, including manual X.509 provisioning with IoT Hub and DPS with individual enrollment do not support automatic device identity certificate renewal.
```

Reply:
Ok, good to know. I copied from another page where it had the params...

https://github.com/MicrosoftDocs/azure-docs/blob/04ab5661a818a56f060545e65933aa45a06e4f99/articles/iot-edge/how-to-manage-device-certificates.md?plain=1#L579

Reply:
Ha! Interesting. Looks like they did a bulk edit to change from site relative to file relative. They should have removed the query parameter. It doesn't matter. It's ignored. Thanks for letting me know.